clang/include/clang/Basic/Attr.td

```diff
@@ -1629,6 +1629,8 @@ def IFunc : Attr, TargetSpecificAttr<TargetELF> {
 
 def Restrict : InheritableAttr {
   let Spellings = [Declspec<"restrict">, GCC<"malloc">];
+  let Args = [IdentifierArgument<"Deallocator", /*opt*/ 1>,
+              ParamIdxArgument<"DeallocatorPtrArgIndex", /*opt*/ 1>];
```

Comment:
```suggestion
  let Args = [IdentifierArgument<"Deallocator", /*opt=*/1>,
              ParamIdxArgument<"DeallocatorPtrArgIndex", /*opt=*/ 1>];
```

Reply:
Nit: Unfortunately we have not been consistent through our codebase in argument comment format but e should strive to be consistent with [bugprone-argument-comment](https://clang.llvm.org/extra/clang-tidy/checks/bugprone/argument-comment.html)

Reply:
:+1: Fixed in b7e125882fe895f0c4fb346a5ae1acddb70f6739